library/test/platforms/android/res/values/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<!-- This file contains resource definitions for displayed strings, allowing
+     them to be changed based on the locale and options. -->
+
+<resources>
+    <!-- Simple strings. -->
+    <string name="app_name">VpLibrary_test_api_create_device_android</string>
```

Comment:
This file, by its comments, seems intended to allow strings to change by locale.  We don't really care about that... is it feasible at all to remove this file and just use a constant string somewhere, or is something like this required for all Android builds?

Reply:
Just did some research and this was tied to the 'label' attribute, which is needed for a short text string associated with an icon on the device.  Since we are only running the APKs through ADB I was able to remove these files and references to them and things seem to be working.  Sweet.